luaui/Widgets/gui_top_bar.lua

```diff
@@ -773,8 +862,12 @@ local function updateResbar(res)
 		local texSize = math_floor(iconSize * 2)
 		if res == 'metal' then
 			glTexture(":lr" .. texSize .. "," .. texSize .. ":LuaUI/Images/metal.png")
-		else
-			glTexture(":lr" .. texSize .. "," .. texSize .. ":LuaUI/Images/energy.png")
+		elseif res == 'energy' then
+
+			glTexture(":lr" .. texSize .. "," .. texSize .. ":LuaUI/Images/energy.png") 
+        else
+            glColor(0.8, 1, 0.8, 1)
```

Comment:
check if this is in the correct place (whitespace suggests a mistake, perhaps just fix whitespace)